spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/Split.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.core.dsl;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The AST node representing a split. A split is a series of jobs to execute in parallel. Those jobs
+ * can themselves be individual jobs or further splits. In DSL form a split is expressed like this:
+ * <pre><tt><aa || bb></tt></pre>.
+ *
+ * @author Andy Clement
+ */
+public class Split extends LabelledComposedTaskNode {
```

Comment:
Done. It crossed my mind when I created it - but I couldn't get particularly excited about making the switch - however, as you commented on it, I'll do it for Split/Flow/TaskApp. I didn't really see it as inconsistent as none of the other subtypes had Node on them.